bundle/src/projects/commercial/modules/dfp/fill-advert-slots.ts

```diff
@@ -30,6 +31,11 @@ const decideAdditionalSizes = (adSlot: HTMLElement): SizeMapping => {
 		return {
 			desktop: [adSizes.billboard],
 		};
+	} else if (contentType === 'LiveBlog' && name?.includes('inline')) {
```

Comment:
This is just a nice to have but wondering if we could update the `contentType` type to be a string literal of the values we expect.

Reply:
The `contentType` type is used in quite a lot of places and in multiple projects and I couldn't find anywhere that provides a definitive list of all possible values. It looks to be non-trivial to update this type to be more specific than `string` so I'll leave it as is for now.